src/altinn-app-frontend/src/shared/resources/dataLists/index.d.ts

```diff
@@ -47,3 +47,46 @@ export interface ISetSort {
   sortColumn: string;
   sortDirection: SortDirection;
 }
+
+export interface IDataList {
+  listItems: any;
+  _metaData: IDataListPaginationData;
+}
+
+export interface IDataLists {
+  [key: string]: IDataListData;
+}
+
+export interface IDataListActualData {
+  listItems: any;
+}
+
+export interface IDataListsMetaData {
+  id: string;
+  mapping?: IMapping;
+  loading?: boolean;
+  secure?: boolean;
+  size?: number;
+  pageNumber?: number;
+  paginationData?: IDataListPaginationData;
+  sortColumn?: string;
+  sortDirection?: SortDirection;
+  dataListId?: string;
+}
+
+export interface IDataListPaginationData {
+  totaltItemsCount: number;
+}
+
+export type IDataListData = IDataListActualData & IDataListsMetaData;
+
+export interface IFetchSpecificDataListSaga {
+  id: string;
+  dataListId: string;
+  formData?: IFormData;
+  language?: string;
+  dataMapping?: IMapping;
+  secure?: boolean;
+  instanceId?: string;
+  paginationDefaultValue?: number;
```

Comment:
`formData`, `language` and `instanceId` seem to never be read (or set) here, and they seem a bit misplaced to me.